### ISSUE TITLE
Remove activity and fragment instantiation tests

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-orga-app#686]
**Is it a ?**

- [ ] Bug Report

- [ ] Feature Request
    
- [x] Chore

### Chore

**Description**

The activity and fragment instantiation tests are flaky and reak with every upgrade of tools. They are used to test the dependency injection and are surely first line of defence to know if the app crashes without any user interaction and the wiring of the app is correct. They also provide a lot of coverage to the project but having flaky tests is worse than having no tests, so until we have a good way to test the instantiation of activities and fragments, they should be disabled for good

**Comment by iamareebjamal** [comment]
@codesun @Masquerade0097 @mishuvs This is a very easy issue. Please someone take it because it is flaky and needs to be resolved first

Also, we have a lot of standing issues, please take up old ones before starting a new issue unless it is a critical bug.

**Comment by mishuvs** [comment]
Alright, I'll work on this.

**Comment by iamareebjamal** [comment]
No one took this issue even when it is mentioned this is easy? Come on guys, this is handing cake in your hands. Please consider existing issues before creating new ones

@srv-twry @Masquerade0097 @mishuvs @DhruvamSharma @KumarBusem @SahanMonaara @sridharjajoo  @Prashant-Chaurasia @YashishDua

**Comment by iamareebjamal** [comment]
Don't close, just take this up. We need to reduce the issues and fasten the issue create-pr sent cycle
